packages/devtools_flutter/test/screen_test.dart

```diff
@@ -0,0 +1,39 @@
+// This is a basic Flutter widget test.
+//
+// To perform an interaction with a widget in your test, use the WidgetTester
+// utility that Flutter provides. For example, you can send tap and scroll
+// gestures. You can also use WidgetTester to find child widgets in the widget
+// tree, read text, and verify that the values of widget properties are correct.
+
+import 'package:devtools_flutter/src/screen.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'wrappers.dart';
+
+void main() {
+  const contentKey = Key('Page Content');
+  const content = SizedBox(key: contentKey);
+
+  group('Page widget', () {
+    testWidgets('displays in narrow mode without error',
+        (WidgetTester tester) async {
+      await setWindowSize(const Size(800.0, 1200.0));
+
+      await tester.pumpWidget(wrap(const Screen(child: content)));
+      expect(find.byKey(contentKey), findsOneWidget);
+      expect(find.byKey(Screen.narrowWidth), findsOneWidget);
+      expect(find.byKey(Screen.fullWidth), findsNothing);
+    });
```

Comment:
nit: trailing comma after `}` here and elsewhere

Reply:
In test methods, the trailing comma doesn't improve the readability, and it only indents the body of the test method by an extra two spaces.

Also, no existing test code does this.